src/common/component-utils/utils.ts

```diff
@@ -10,9 +10,10 @@ export function excludeComponent(nodes: ReactNode = [], componentType: FC): Reac
     return elements.filter(({ type }) => type !== componentType)
 }
 
-export function filterByType(nodes: ReactNode = [], componentType: FC, fn?: () => void): ReactElement[] {
+export function filterByType(nodes: ReactNode = [], componentType: FC | FC[], fn?: () => void): ReactElement[] {
     const elements = Children.toArray(nodes) as ReactElement[]
-    return elements.filter(({ type }) => type === componentType)
+    const types = [componentType].flat()
```

Comment:
Sorry, I just notice this later but unfortunately, array flat is still not supported on mobile Safari 11. It is available starting from Safari 12, [see browser support for array.flat()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/flat#browser_compatibility). According to eBay Browser Policy, mobile Safari 11 is categorized as Tier 2 browsers but we still need support it for critical sites.

See:

- eBay Browserlist config: https://github.com/eBay/browserslist-config/blob/v2.4.1/index.js#L8
- See eBay internal wiki about browser policy

We need to manually check & unwrap `FC[]`, or use arrayflat polyfill.